tests/commands/config.test.js

```diff
@@ -0,0 +1,17 @@
+const {execSync} = require('child_process');
+const EXAMPLE_API_KEY = '35aa457f06e1da94f9b37420c56f5909';
```

Comment:
For me it's not okey. 

Better add it by env variable.

And don't push API KEY in PR

Reply:
Thanks for the comment, to clarify this is not a real API key (it was not generad by flotiq). I hardcoded it, becouse it is inside a test case and env was not used in this repo yet. Do you think we should switch to it?

Reply:
Yes, I think so. 
In my PR I started with this - env variable with host of API. 
And than we can run this test with secret in GH